samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/CommonMessages_es.properties

```diff
@@ -55,3 +55,4 @@ cancel_question_info_no_undo=Esta acci\u00f3n no puede deshacerse!
 cancel_question_info_skip_question=Puedes omitir esta pregunta.
 cancel_question_error_cancelling=Ha ocurrido un error al intentar cancelar la pregunta. No se han realizado cambios.
 cancel_question_info_emi=En las preguntas de tipo Relacionar M\u00faltiples Opciones no se ajustar\u00e1 la puntuaci\u00f3n.
+cancel_question_info_random=Questions of assessments with questions that are randomly assigned from question pools can not be cancelled.
```

Comment:
@NuriaQS, could you help with this translation, please?

Reply:
cancel_question_info_random=Las preguntas de las evaluaciones con preguntas asignadas aleatoriamente a partir de grupos de preguntas no pueden anularse.

Reply:
I think the Spanish translations use "exámenes" for assessments.  Does `cancel_question_info_random=Las preguntas de las ex\u00e1menes con preguntas asignadas aleatoriamente a partir de grupos de preguntas no pueden anularse.` look right?